src/Welcome/EthosPage.jsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { Row, Col } from "../../src/index";
 import WelcomeStyles from "./welcome.style";
+import header from "../../public/images/header.png";
```

Comment:
is this being used?

Reply:
I removed to keep the page looking the same